lib/commands/updateDelaySet-4.lua

```diff
@@ -21,7 +24,25 @@ if (score ~= nil) then
   if (math.floor(score) <= tonumber(ARGV[2])) then
     redis.call("ZREM", KEYS[1], jobId)
     redis.call("LREM", KEYS[2], 0, jobId)
-    redis.call("LPUSH", KEYS[3], jobId)
+
+    local priority = tonumber(redis.call("HGET", ARGV[1] .. jobId, "priority"))
+
```

Comment:
I am a bit concerned about what happens if "priority" is not defined, for example in the case that you have a queue already and you upgrade bull, so that some old jobs where added without the priority field. Will ```tonumber``` coerce to 0?

Reply:
ok, I read the documentation of tonumber :) and it seems that it will return ```nil``` if the number could not be converted, so the if condition should be changed to:
```lua
if priority ~= nil or priority == 0 then
```

Reply:
i think you meant `priority == nil`
nevertheless, i moved it to the variable assignment `local priority = tonumber(...) or 0`
i think it is more clean that way